### DISCUSSION TITLE
Gradle 网络问题的一种解决方法 / A solution for network issues with Gradle

### DISCUSSION THREAD

**Discussion by wangyiwei2015** [skiptools/skip#110] [Show and tell]
As the access to global internet is partially blocked in China, a proxy to bypass the GFW is necessary for almost all developers. In this case, development with Skip Tools needs some tricks to work properly. This is unlikely to happen in other regions that have normal internet access. 💩

如果跟着官方教程干净地创建 Skip 工程，那么在 Gradle 执行阶段遇到的问题，很可能是网络错误导致（表现为 Xcode 构建时的报错是 `gradle -p ../Android launchDebug` 命令运行失败）。

此时，挂梯子，进终端，配置终端走代理（这一步如果不理解可以自己搜一下）。`cd` 到 Skip 工程目录的 Android 文件夹，执行 `gradle launchDebug` ，它就会下载所需依赖。成功之后再回到 Xcode 去构建或运行 App。

如果怀疑生成目录不干净，可以进去 `gradle clean` 一下。

**Comment by aabewhite** [comment]
Thanks! That's good to know
